# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libchipcard2/hotfix_sysfsutils.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur libchipcard2-1.9.20beta-orig/src/lib/chipcard2-server/common/usbttyscanner.c libchipcard2-1.9.20beta/src/lib/chipcard2-server/common/usbttyscanner.c
--- libchipcard2-1.9.20beta-orig/src/lib/chipcard2-server/common/usbttyscanner.c	2005-10-22 00:45:35.000000000 +0000
+++ libchipcard2-1.9.20beta/src/lib/chipcard2-server/common/usbttyscanner.c	2006-02-11 20:11:58.659535500 +0000
@@ -110,7 +110,7 @@
                         struct sysfs_device) {
       /* for each device: look for ttyUSB in children*/
       port = -1;
-      temp_device = sysfs_open_device_tree(curdev->path);
+      temp_device = sysfs_open_device_path(curdev->path);
       if (temp_device) {
         if (temp_device->children) {
           dlist_for_each_data(temp_device->children, child,
